README.md

```diff
@@ -149,8 +149,6 @@ Example:
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your Future Feature section more descriptive and professional by adding some more exciting features 👍

Reply:
Thanks you for your suggestion, I will do it!